tests/test_interfaces.py

```diff
@@ -7,7 +7,7 @@
 from testapp.factories import BlogPageFactory, CustomInterfaceBlockFactory
 from testapp.interfaces import CustomInterface
 
-from grapple.types.pages import PageInterface, get_page_interface
+from grapple.types.interfaces import PageInterface, get_page_interface
```

Comment:
can you update `testapp.interfaces.CustomInterface` to inherit from `PageInterface`?

we have `test_get_page_interface_with_custom_page_interface` but it was not fully testing your scenario

Reply:
I've created a new interface, `CustomPageInterface`, for this purpose; `CustomInterface` is applied to types that are not pages and so things break.